src/Symfony/Bundle/WebProfilerBundle/Resources/views/Router/panel.html.twig

```diff
@@ -15,7 +15,7 @@
             <p>No parameters.</p>
         </div>
     {% else %}
-        {{ include('@WebProfiler/Profiler/table.html.twig', { data: request.routeParams, labels: ['Name', 'Value'] }, with_context = false) }}
+        {{ include('@WebProfiler/Profiler/table.html.twig', {data: request.routeParams, labels: ['Name', 'Value']}, with_context: false) }}
```

Comment:
this requires making web-profilder-bundle conflict with twig version that don't support the new notation for named arguments.

Reply:
@stof reading https://github.com/twigphp/Twig/blob/3.x/CHANGELOG, you mean I need tu upgrade to twig 3.15+ is that correct?

Reply:
I've done the same for security bundle were similar changes occurred